locale/fr.json

```diff
@@ -0,0 +1,2122 @@
+{
+    "login": {
+        "copy": {
+            "translation": "Si vous avez déjà un compte human-connection, connectez-vous ici.",
+            "notes": ""
+        },
+        "login": {
+            "translation": "Connexion",
+            "notes": ""
+        },
+        "logout": {
+            "translation": "Déconnexion",
+            "notes": ""
+        },
+        "email": {
+            "translation": "Votre courriel",
```

Comment:
"Votre courriel" seems a bit strange, I think it should be "Votre mail" or "Votre addresse mail"